### DISCUSSION TITLE
Allow locale when using sms authentication

### DISCUSSION THREAD

**Discussion by andreacab2** [supabase/supabase#17729] [Feature Requests]
Hi there, 
We use a passwordless strategy to authenticate our users with Twilio Verify but we cannot see how to change the locale of our SMS sent to our users. We checked the supabase docs and [here](https://supabase.com/docs/guides/auth/phone-login/twilio#sms-custom-template) it says:

> SMS custom template
The SMS message sent to a phone containing an OTP code can be customized. This is useful if you need to mention a brand name or display a website address.
Go to Auth > Providers page (under "Configuration") in the Supabase dashboard and select "Phone" from the Auth Providers list (https://supabase.com/dashboard/project/_/auth/providers). Scroll to the very bottom of the "Phone" section to the "SMS Message" input - you can customize the SMS message here.

But we cannot find this anywhere in the supabase dashboard.

Next we looked at the [Twillio Verify API](https://www.twilio.com/docs/verify/api/verification#start-new-verification) and it does allow to pass in a locale to the request in order to change the locale of the SMS. Is this right? And if so, is anyone working on improving that? If not, we can definitely contribute and help!

Thanks for the help and info AND the awesome tool that supabase is 👌 💯

**Comment by code-of-tack** [comment]
I have the same issue. were you able to fix it?

**Comment by zalmanlew** [comment]
I have the same issue, it's really annoying.

It should allow passing it in the `options` object like
```js
supabase.auth.signInWithOtp({ phone, options: { locale: "en" } });

```

https://www.twilio.com/docs/verify/supported-languages

**Comment by vbelopolskiy** [comment]
Any news on this question?

**Comment by balajsebastian** [comment]
Any news on this?
